docs/docs/advanced.md

```diff
@@ -364,6 +364,140 @@ If you get a `403 Forbidden` error when testing the plugin, make sure you correc
 
 ## Recipes
 
+### What are recipes?
+
+In Xen Orchestra, recipes are ready-to-use automation templates that make it easy to deploy complex infrastructures. You don’t need to configure each virtual machine manually. 
+
+With a few clicks, you can launch a full multi-VM environment, where all nodes are automatically set up and connected.
```

Comment:
```suggestion
With a few clicks, you can launch a full multi-VM environment, where all VMs are automatically set up and connected.
```

nodes could be understood as physical host : use VM to avoid disambiguate here